test/cpp/api/CMakeLists.txt

```diff
@@ -40,15 +42,10 @@ set(TORCH_API_TEST_SOURCES
   ${TORCH_API_TEST_DIR}/grad_mode.cpp
   ${TORCH_API_TEST_DIR}/operations.cpp
 )
-if(USE_CUDA)
+if(USE_CUDA OR USE_ROCM)
```

Comment:
The `if()` can be removed and merged to the previous `set()` statement.

Reply:
> The `if()` can be removed and merged to the previous `set()` statement.

what happens for CPU only build? putting it to set() will be problem for those build